layouts/default.vue

```diff
@@ -30,6 +31,7 @@ export default {
 
 .content {
   min-height: 100vh;
+  margin-bottom: 0px !important;
```

Comment:
For some reason Bulma is injecting bottom margins. This revises that and removes the margins